docs/mkdocs/docs/index.md

```diff
@@ -1,387 +1,495 @@
-- **Flexible**: Supporting data with and without a schema, ArcticDB is also fully compatible with streaming data ingestion. The platform is bitemporal, allowing you to see all previous versions of stored data.
-- **Familiar**: ArcticDB is the world's simplest database, designed to be immediately familiar to anyone with prior Python and Pandas experience.
-
-#### What is ArcticDB _not_?
-
-ArcticDB is designed for high throughput analytical workloads. It is _not_ a transactional database and as such is not a replacement for tools such as PostgreSQL.
+- **Fast**
+    - Process millions of rows per second
+    - Quick and easy to install: `pip install arcticdb`
+- **Flexible**
+    - No data schema is needed
+    - Supports streaming data ingestion
+    - Bitemporal - stores all previous versions of stored data
+    - As easy to get started as using files
+    - Scales from dev/research to production environments
+- **Familiar**
+    - ArcticDB is the world's simplest database
+    - Easy to learn for anyone with Python and Pandas experience
+    - Just you and your data - the cognitive overhead is very low.
+
+#### Transactions: Are They Overrated?
+
+- ArcticDB is designed for high throughput analytical workloads
+- It is not a transactional database
+- Most analytical workflows can be constructed to run without needing transactions
+- So why pay the cost of transactions when they are often not needed?
 
 ## Getting Started
 
-The below guide covers installation, setup and basic usage. More detailed information on advanced functionality such as _snapshots_ and _parallel writers_ can be found in the tutorials section.
+This section will cover installation, setup and basic usage. More details on basics and advanced features can be found in the [tutorials](tutorials/fundamentals.md) section.
 
 ### Installation
 
 ArcticDB supports Python 3.6 - 3.11. To install, simply run:
 
-```
+```python
 pip install arcticdb
 ```
 
 ### Setup
 
-ArcticDB is a storage engine designed for object storage, but also supports local-disk storage using LMDB.
+ArcticDB is a storage engine designed for object storage and also supports local-disk storage using LMDB.
 
 !!! Storage Compatibility
 
-    ArcticDB supports any S3 API compatible storage. It has been tested against AWS S3 and storage appliances like [VAST Universal Storage](https://vastdata.com/).
+    ArcticDB supports any S3 API compatible storage, including AWS and Azure, and storage appliances like [VAST Universal Storage](https://vastdata.com/) and [Pure Storage](https://purestorage.com/).
 
     ArcticDB also supports LMDB for local/file based storage - to use LMDB, pass an LMDB path as the URI: `adb.Arctic('lmdb://path/to/desired/database')`.
 
 To get started, we can import ArcticDB and instantiate it:
 
 ```python
->>> import arcticdb as adb
->>> ac = adb.Arctic(<URI>)
+import arcticdb as adb
+uri = "insert your uri string here"
```

Comment:
I would put a realistic looking uri here.
It really helps drive the point of how easy it is to setup Arctic